tags/tags.go

```diff
@@ -11,7 +11,7 @@ const (
 	TagConfigServer   = prefixLibTag + "server"
 	TagHashiCorpVault = prefixLibTag + "vault"
 
-	AmountTags = 7
+	AmountTags = 8
```

Comment:
You have not added anything here, why do you increase the number?

Reply:
Max index for last tag = 7. If i will create array for len = 7 program will exit with panic. This need for logic part programm. I can not be changed this